packages/sitecore-jss-angular/.gitignore

```diff
@@ -1,3 +1,4 @@
 # compiled output
 .ng_build
 coverage
+.angular
```

Comment:
What does this folder have and why it's generated?

Reply:
it's a build cache - turned on by default, but we can disable it, if you think it's neccessary